src/sequences/SequenceEntity.js

```diff
@@ -21,15 +22,15 @@ export class SequenceEntity extends Entity {
 		const subSequences = this._subSequences();
 
 		subSequences.forEach((entity) => {
-			entity && entity.href && this._subEntity(SequenceEntity, entity.href, onChange);
+			entity && this._subEntity(SequenceEntity, entity, onChange);
 		});
 	}
 
 	onSequencedActivityChange(onChange) {
 		const sequencedActivity = this._sequencedActivity();
 
 		sequencedActivity.forEach((entity) => {
-			entity && entity.href && this._subEntity(SequencedActivityEntity, entity.href, onChange);
+			entity && this._subEntity(SequencedActivityEntity, entity, onChange);
```

Comment:
This will work if the entity is a complete entity or has a link to the complete entity

Reply:
what do you mean by complete entity here?